lib/foreman_discovery/engine.rb

```diff
@@ -66,7 +66,7 @@ class Engine < ::Rails::Engine
             :discovered_hosts          => [:update_multiple_location,
                                            :select_multiple_organization,
                                            :update_multiple_organization,
-                                           :select_multiple_location, :refresh_facts, :reboot, :reboot_all],
+                                           :select_multiple_location, :refresh_facts, :reboot, :multiple_reboot],
             :hosts                     => [:process_hostgroup],
             :"api/v2/discovered_hosts" => [:refresh_facts, :reboot, :reboot_all]
           }, :resource_type => 'Host'
```

Comment:
This is missing some actions, I don't understand why I dont see failures locally but on jenkins:

AccessPermissionsTest.test_0739_route discovered_hosts/submit_multiple_reboot should have a permission that grants access
AccessPermissionsTest.test_0740_route discovered_hosts/submit_multiple_auto_provision should have a permission that grants access

http://ci.theforeman.org/job/test_plugin_matrix/5117/database=postgresql,ruby=2.4,slave=fast/

Reply:
Oh it does fail locally, cool. Let me fix this.

Reply:
Merged as 84d8e45